Data_Structures.py

```diff
@@ -72,8 +74,14 @@ class Cell:
     def __init__(self, x, y, dist, prev):
         self.x = x
         self.y = y
-        self.dist = dist;  # distance to start
-        self.prev = prev;  # parent cell in the path
+        self.dist = dist  # distance to start
+        self.prev = prev  # parent cell in the path
+
+    def X(self):
```

Comment:
why separate it into 2 functions? return a position (x,y), and  the caller will use whatever he wants